lib/constants.ts

```diff
@@ -1,13 +1,32 @@
 export const OZONE_SERVICE_DID =
   process.env.NEXT_PUBLIC_OZONE_SERVICE_DID || undefined
 
+export const OZONE_PUBLIC_URL =
+  process.env.NEXT_PUBLIC_OZONE_PUBLIC_URL ||
+  (process.env.NODE_ENV === 'development'
+    ? 'http://localhost:2587'
+    : // Defaults to "window.location.origin"
+      undefined)
+
 export const PLC_DIRECTORY_URL =
-  process.env.NEXT_PUBLIC_PLC_DIRECTORY_URL || `https://plc.directory`
+  process.env.NEXT_PUBLIC_PLC_DIRECTORY_URL ||
+  (process.env.NODE_ENV === 'development'
+    ? 'http://localhost:2582'
+    : 'https://plc.directory')
 
 export const QUEUE_CONFIG = process.env.NEXT_PUBLIC_QUEUE_CONFIG || '{}'
 
-export const SOCIAL_APP_DOMAIN = 'bsky.app'
-export const SOCIAL_APP_URL = `https://${SOCIAL_APP_DOMAIN}`
+export const SOCIAL_APP_URL =
+  process.env.NEXT_PUBLIC_SOCIAL_APP_URL ||
+  (process.env.NODE_ENV === 'development'
+    ? 'http://localhost:2584'
+    : 'https://bsky.app')
+
+export const HANDLE_RESOLVER_URL =
+  process.env.NEXT_PUBLIC_HANDLE_RESOLVER_URL ||
+  (process.env.NODE_ENV === 'development'
+    ? 'http://localhost:2584'
+    : 'https://bsky.social')
```

Comment:
The endpoint on the appview is going to be more reliable for this purpose.  Perhaps we can eventually implement this endpoint on the Ozone backend and it can serve itself.
```suggestion
export const HANDLE_RESOLVER_URL =
  process.env.NEXT_PUBLIC_HANDLE_RESOLVER_URL ||
  (process.env.NODE_ENV === 'development'
    ? 'http://localhost:2584'
    : 'https://api.bsky.app')
```